viewer/packages/tools/src/Measurement/Line.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+import * as THREE from 'three';
+import { LineGeometrySegment } from './LineGeometrySegment';
+
+export class Line extends THREE.Mesh {
```

Comment:
Do we really need our own implementation for this? Isn't Line3 from ThreeJS enough to represent the line? Cant https://threejs.org/docs/?q=Line#api/en/objects/Line be used to render it? Am I missing something here?

Reply:
Use GL_LINES, will not be possible to add line width or any other style if required